go/tools/go-upgrade/go-upgrade.go

```diff
@@ -162,13 +150,9 @@ func init() {
 	getCmd.AddCommand(getGoCmd)
 	getCmd.AddCommand(getBootstrapCmd)
 
-	upgradeCmd.AddCommand(upgradeWorkflowsCmd)
-
 	upgradeCmd.Flags().BoolVar(&workflowUpdate, "workflow-update", workflowUpdate, "Whether or not the workflow files should be updated. Useful when using this script to auto-create PRs.")
```

Comment:
`--workflow-update` can also be removed since we won't be upgrading the golang version in the workflows.

Reply:
@frouioui makes sense, removed that too!